Packs/AzureNetworkSecurityGroups/Integrations/AzureNetworkSecurityGroups/README.md

```diff
@@ -1,59 +1,134 @@
+   | Application ID                     |                                                                                                                                                                                                    | False |
+   | Default Subscription ID            | There are two options to set the specified value, either in the configuration or directly within the commands. However, setting values in both places will cause an override by the command value. | True |
+   | Default Resource Group Name        | There are two options to set the specified value, either in the configuration or directly within the commands. However, setting values in both places will cause an override by the command value. | True |
+   | Azure AD endpoint                  | Azure AD endpoint associated with a national cloud.                                                                                                                                                | False |
+   | Trust any certificate (not secure) |                                                                                                                                                                                                    | False |
+   | Use system proxy settings          |                                                                                                                                                                                                    | False |
+   | Authentication Type                | Type of authentication - can be Authorization Code flow \(recommended\), Client Credentials, Device Code Flow, or Azure Managed Identities.                                                        | True |
+   | Tenant ID                          |                                                                                                                                                                                                    | False |
+   | Client Secret                      |                                                                                                                                                                                                    | False |
+   | Application redirect URI           |                                                                                                                                                                                                    | False |
+   | Authorization code                 | For user-auth mode - received from the authorization step. See Detailed Instructions \(?\) section.                                                                                                | False |
+   | Azure Managed Identities Client ID | The Managed Identities client ID for authentication - relevant only if the integration is running on Azure VM.                                                                                     |False |
```

Comment:
Not sure why you edited this lines.  It will not render correctly. As far as I can tell, it was fine the way it was.